lib/chai/core/assertions.js

```diff
@@ -1151,6 +1183,7 @@ module.exports = function (chai, _) {
    * @api public
    */
 
+  // Should this also be updated? + assertMost and within
```

Comment:
Originally we discussed only `isAbove` and `isBelow` methods to support dates, but this seems connected as well. Same question goes for assertMost and within. [restored]